src/main/frontend/persist_db/browser.cljs

```diff
@@ -77,9 +77,13 @@
       (.releaseAccessHandles sqlite repo)))
 
   (<transact-data [_this repo tx-data tx-meta]
-    (let [^js sqlite @*sqlite]
-      (p/let [_ (when sqlite (.transact sqlite repo (pr-str tx-data) (pr-str tx-meta)))]
-        nil)))
+    (let [^js sqlite @*sqlite
+          tx-data' (pr-str tx-data)
+          tx-meta' (pr-str tx-meta)]
+      (p/do!
+       (ipc/ipc :db-transact repo tx-data' tx-meta')
+       (when sqlite (.transact sqlite repo tx-data' tx-meta'))
```

Comment:
This isn't a change in this PR but if sqlite is missing, we should fail explicitly to the user so users are aware their data isn't being written and they could lose a lot of data silently. I can address this tomorrow if you'd like

Reply:
Agreed! Thanks for the help!